android/build.gradle

```diff
@@ -131,8 +131,8 @@ dependencies {
   api 'com.facebook.react:react-native:+'
 
   implementation 'androidx.core:core-ktx:1.8.0'
-  releaseImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk:v3.2.2'
-  debugImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk-debug:v3.2.2'
+  releaseImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk:v3.3.0'
+  debugImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk-debug:v3.3.0'
```

Comment:
I think we will need to pin it to 3.3.1 after we release the Android FPJS throttling work, just FYI

Reply:
will updated this after the ADV check ENG-7894 is merged.